src/Libraries/Microsoft.Extensions.AI.Abstractions/ChatCompletion/ChatOptions.cs

```diff
@@ -9,6 +9,16 @@ namespace Microsoft.Extensions.AI;
 /// <summary>Represents the options for a chat request.</summary>
 public class ChatOptions
 {
+    /// <summary>Gets or sets an optional identifier used to associate a request with an existing conversation.</summary>
+    /// <remarks>This property is obsolete. Use <see cref="ConversationId"/> instead.</remarks>
+    [System.Obsolete("Use ConversationId instead.")]
+    [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
+    public string? ChatThreadId
+    {
+        get => ConversationId;
+        set => ConversationId = value;
+    }
```

Comment:
There are also corresponding `ConversationId` properties on `ChatResponse` and `ChatResponseUpdate` so I think they would need corresponding fallbacks.